odoo_backup_sh/models/odoo_backup_sh.py

```diff
@@ -41,7 +41,12 @@ class BackupConfig(models.Model):
     _description = 'Backup Configurations'
     _rec_name = 'database'
 
-    DATABASE_NAMES = [(db, db) for db in odoo.service.db.list_dbs() if db != 'session_store']
+    if config['list_db']:
+        DATABASE_NAMES = [(db, db) for db in odoo.service.db.list_dbs() if db != 'session_store']
+    elif config['db_name']:
+        DATABASE_NAMES = [(db, db) for db in odoo.tools.config['db_name'].split(',') if db != 'session_store']
```

Comment:
exception for session_store can be deleted as it's from abandoned docker deployment

Reply:
Using odooup I still have `list_dbs()` array with 'session_store' value.

Reply:
ok